app.yml

```diff
@@ -0,0 +1,33 @@
+.aliases:
+  core: &core
+    port: 8080
+    public:
+      ingress: true
+    scale:
+      horizontal: 3-12
+      vertical: large
+      cpuThreshold: 30
+    healthcheck:
+      path: /
+      liveness:
+        initialDelaySeconds: 15
+        timeoutSeconds: 30
+      readiness:
+        initialDelaySeconds: 15
+
+engine: ~1
+name: covidflow
+components:
+  core-en:
+    <<: *core
+  core-fr:
+    <<: *core
+scale:
+  memThreshold: false
+environments:
+  dev-ca2:
+    environment:
+      COUNTRY_CODE: ca
```

Comment:
If this variable is required for all environment you can set it at root level

```
environment:
  COUNTRY_CODE: ca
```

Reply:
merci!